requirements/edx/base.txt

```diff
@@ -361,7 +362,7 @@ django-storages==1.8
     #   edxval
 django-user-tasks==3.0.0
     # via -r requirements/edx/base.in
-django-waffle==2.4.1
+django-waffle==2.5.0
```

Comment:
No breaking change mentioned in the [CHANGELOG](https://github.com/django-waffle/django-waffle/blob/master/CHANGES#L6).